config/main.py

```diff
@@ -2166,6 +2166,23 @@ def delete(ctx):
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
 
+#
+# 'feature' command ('config feature name state')
+# 
+@config.command('feature')
+@click.argument('name', metavar='<feature-name>', required=True)
+@click.argument('state', metavar='<enabled/disabled>', required=True, type=click.Choice(["enabled", "disabled"]))
```

Comment:
Please change metavar value to reflect usage instead of possible values ( `<feature-state>`?), now that possible values are handled by click.Choice.